doc/installation/system/inifile.rst

```diff
@@ -78,9 +78,23 @@ You can set ``PI_UI_DEACTIVATED = True`` to deactivate the privacyIDEA UI.
 This can be interesting if you are only using the command line client or your
 own UI and you do not want to present the UI to the user or the outside world.
 
-.. note:: The API calls are all still accessable, i.e. privacyIDEA is
+.. note:: The API calls are all still accessible, i.e. privacyIDEA is
    technically fully functional.
 
+
+.. _engine-registry:
+
+Engine Registry Class
+---------------------
+
+The ``PI_ENGINE_REGISTRY_CLASS`` option controls the pooling of database connections
+opened by SQL resolvers and the audit module. If it is set to ``"null"``,
```

Comment:
...and the **SQL** Audit module...

there coude be(TM) be other audit modules.